CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ This is fixed now by ignoring the sampleRate for transactions. If you use custom
 - fix: Possible endless loop for onCrashedLastRun (#1734)
 - fix: Wrongly sampling transactions (#1716)
 - feat: Add flag for UIViewControllerTracking (#1711)
+- feat: Error Screenshots (#1751)
```

Comment:
Should go under `Unreleased`

Reply:
I think this was a merging problem. I will fix it.